brightwind/transform/transform.py

```diff
@@ -198,7 +196,7 @@ def _round_timestamp_down_to_averaging_prd(timestamp, period):
                                              hour=timestamp.hour)
     elif period[-1] == 'M' or period[-2:] == 'MS':
         return '{year}-{month}'.format(year=timestamp.year, month=timestamp.month)
-    elif period[-2:] == 'AS' or period[-1:] == 'A':
+    elif period[-2:] == 'YS' or period[-1:] == 'A':
```

Comment:
Should the 'A' be 'Y' too?

If users already use 'AS' or 'A' then wouldn't this be a breaking change? Could we include 'YS' or 'Y' as well as keeping the 'AS' and 'A'?